builtin/mainmenu/tab_local.lua

```diff
@@ -357,6 +357,40 @@ local function main_button_handler(this, fields, name, tabdata)
 		local game_obj
 		if world then
 			game_obj = pkgmgr.find_by_gameid(world.gameid)
+			if not game_obj then
+				local games = pkgmgr.games
+				for i=1, #games do
+					local aliases = games[i].aliases:split()
+					for _, alias in pairs(aliases) do
+						alias = alias:trim()
+						if alias == world.gameid then
+							game_obj = games[i]
+							world.gameid = games[i].id
+							break
+						else
+							local el_id_len = #world.gameid
+							if el_id_len > 5 and world.gameid:sub(el_id_len - 4) == "_game" then
```

Comment:
The normalization code again :upside_down_face: this is effectively duplicating the logic in the "filter function"? Why?